UI/index.erb

```diff
@@ -141,5 +101,13 @@
             </div>
         </div>
         <div id="isMobile"></div>
+        <script>
+          (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+          (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+          m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+          })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+          ga('create', 'UA-70922164-1', 'auto');
```

Comment:
Should the UA-..... value be a property?

Are we sure that we don't need different values for test/dev/prod?